airflow/models/dag.py

```diff
@@ -1894,6 +1895,15 @@ def set_is_paused(self,
         try:
             for dag_model in dag_models:
                 dag_model.is_paused = is_paused
+            if is_paused:
```

Comment:
Can we get rid of this if block and send an email notifying users whenever its unpaused as well? It goes in both directions, where if they disabled there dag and someone enables it they should know.

Reply:
Sure, I can add that.